### PR TITLE
Add workflow category and items

### PR DESCRIPTION
**Pull request by chocopowwwa**
I hope the title is useful enough 😆

### CODE REVIEW

**Comment by herrbischoff** [comment]
First off: only one suggestion per pull request. Read: [contributing.md](https://github.com/herrbischoff/awesome-osx-command-line/blob/master/contributing.md).

File: README.md [hunk 1]
```diff
@@ -331,6 +332,11 @@ If you want to contribute, you are highly encouraged to do so. Please read the [
 - [racoon](http://ipsec-tools.sourceforge.net) - Internet Key Exchange (IKE) daemon for automatically keying IPsec connections.
 - [strongSwan](https://www.strongswan.org/) - Open Source IPsec for Linux.
 
+### Workflow
+
+- [Corgi](https://github.com/DrakeW/corgi) - Command-line workflow manager that helps with your repetitive command usages by organizing them into reusable snippet.
```
Comment (herrbischoff) [inline README.md:337]: What is the upside to just writing a shell script?

File: README.md [hunk 2]
```diff
@@ -331,6 +332,11 @@ If you want to contribute, you are highly encouraged to do so. Please read the [
...
+- [Pet](https://github.com/knqyf263/pet) - Simple command-line snippet manager.
```
Comment (herrbischoff) [inline README.md:338]: This appears to be a rather bloated and confounded way to run snippets. I'd arguably be quicker in grepping a file with shortcuts, pasting and editing the command and run it.

Comment (chocopowwwa) [reply README.md:338]: hmm yeah, i agree with you, it wasn't necessary to use such tool, i think i'm going to close the PR, thx

**Pull request closed without merging**